### PR TITLE
Ensure policy check doesn't run for remote backend.

### PR DESCRIPTION
**Pull request by nishkrishnan**

### CODE REVIEW

**Comment by nishkrishnan** [comment]
👀  #orchestration @msarvar tests coming shortly. Figured Id see if you have any better ideas.

File: server/events/runtime/runtime.go [hunk 1]
```diff
@@ -67,6 +73,15 @@ func GetPlanFilename(workspace string, projName string) string {
 	return fmt.Sprintf("%s-%s.tfplan", projName, workspace)
 }
 
+// isRemotePlan returns true if planContents are from a plan that was generated
+// using TFE remote operations.
+func IsRemotePlan(planContents []byte) bool {
+	// We add a header to plans generated by the remote backend so we can
+	// detect that they're remote in the apply phase.
+	remoteOpsHeaderBytes := []byte(remoteOpsHeader)
```
Comment (msarvar) [inline server/events/runtime/runtime.go:81]: This is tripping me up, where is `remoteOpsHeader` declared?

**Review by msarvar — DISMISSED** [review]

File: server/events/runtime/runtime.go (hunk 1 above)
Comment (nishkrishnan) [reply server/events/runtime/runtime.go:81]: https://github.com/lyft/atlantis/blob/master/server/events/runtime/plan_step_runner.go#L325:L326

Comment (msarvar) [reply server/events/runtime/runtime.go:81]: oh, weird. Searching didn't return anything.

File: server/events/runtime/apply_step_runner.go [hunk 2]
```diff
@@ -38,7 +37,9 @@ func (a *ApplyStepRunner) Run(ctx models.ProjectCommandContext, extraArgs []stri
 
 	ctx.Log.Info("starting apply")
 	var out string
-	if a.isRemotePlan(contents) {
+
+	// TODO: Leverage PlanTypeStepRunnerDelegate here
+	if IsRemotePlan(contents) {
```
Comment (msarvar) [inline server/events/runtime/apply_step_runner.go:42]: I the plan to do this later? Should we track it in jira?

**Review by msarvar — APPROVED** [review]
LGTM.

File: server/events/runtime/apply_step_runner.go (hunk 2 above)
Comment (nishkrishnan) [reply server/events/runtime/apply_step_runner.go:42]: Yeah this is more cleanup work which I don't want to do now.  Probably ok to not track refactors in JIRA unless they are essential for a feature.

**Pull request merged**